rfcs/shared/build-system/05.1-react-components-folder-organization.md

```diff
@@ -0,0 +1,39 @@
+# RFC: Group `@fluentui/react-components` Packages to Common Folder
+
+---
+
+Contributors: Fluent UI React Build team, @TristanWatanabe
+
+## Summary
+
+This RFC is a follow-up to the [Repo Folder Organization](https://github.com/JustSlone/fluentui/blob/9e46742b4383af09e58365c0deb7bc70a3297261/rfcs/build-system/05-folder-organization.md) RFC which focuses on the contents of a new common folder for `@fluentui/react-components` packages to live in.
+
+## Background
+
+In prep for the stable release of `@fluentui/react-components`, our repo will start to gain more visitors and contributors who will seek `react-components` related source code and information. Having a common folder where all `@fluentui/react-components` components are located will help make this process straightforward.
+
+## Problem statement
+
+As it currently stands, finding `@fluentui/react-components` related components is not straightforward as they currently live alongside `@fluentui/react`, `@fluentui/northstar` and `@fluentui/web-components` components. Simply searching for the `Button` component will yield a plethora of results spanning all 4 libraries within the monorepo. This can lead to confusion amongst contributors who visit our repo.
+
+## Detailed Design or Proposal
+
+1. Within the packages folder, we should create a new subfolder called `react-components` and move all `@fluentui/react-components` packages to it.
```

Comment:
As currently laid out in the RFC proposal, the `react-components` package itself would go into the new react-components subfolder since this new folder is supposed to contain all v9 related packages.